homeassistant/components/media_player/samsungtv.py

```diff
@@ -0,0 +1,175 @@
+"""
+homeassistant.components.media_player.denon
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Provides an interface to Samsung TV with a Laninterface.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/media_player.samsungtv/
+"""
+import logging
+import socket
+from samsungctl import Remote
```

Comment:
Please do this import in a Function or Method and not at the module level. Home Assistant needs to be able to import your platform before its dependencies are installed.

You will also need to add a line with something like the following:
```python
REQUIREMENTS = ['samsungctl==1.0.0']
```
This is what tells Home Assistant about any required packages for this component. Home Assistant will read this first, then go fetch those packages before setting up this component.